internal/website/content/howto/blob/open-bucket.md

```diff
@@ -0,0 +1,328 @@
+---
+title: "Open a Bucket"
+date: 2019-03-20T14:51:29-07:00
+draft: true
+weight: 1
+---
+
+The first step in interacting with unstructured storage is connecting to your
+storage backend. Every storage provider is a little different, but the Go CDK
```

Comment:
Should we use backend vs. provider (vs. driver) consistently? Maybe just "provider" everywhere, since this is the terminology we use elsewhere and "backend" is too overloaded a term

Reply:
Using provider.

Reply:
Ack, already changed.